doc/modules/preprocessing.rst

```diff
@@ -426,6 +426,47 @@ features, one has to explicitly set ``n_values``. For example,
 See :ref:`dict_feature_extraction` for categorical features that are represented
 as a dict, not as integers.
 
+.. _discretization:
+
+Discretization
```

Comment:
Binarization is an instance of discretization. Discretization should appear as a major heading, with a subheading for fixed-width and a subheading for binarization.